keyboards/handwired/replicazeron/common/thumbstick.h

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2023 9R
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "analog.h"
+#include "debug.h"
+#include "math.h"
+#include "quantum.h"
```

Comment:
```suggestion
```

Reply:
if that include is removed I get compile errors:

```
keyboards/handwired/replicazeron/common/thumbstick.c:50:9: error: implicit declaration of function 'register_code16' [-Werror=implicit-function-declaration]
keyboards/handwired/replicazeron/common/thumbstick.c:52:9: error: implicit declaration of function 'unregister_code16' [-Werror=implicit-function-declaration]

```
So I think this needs to stay or is there another, better way to solve this without including quantum.h?